react/components/ProductSummaryPrice/ProductSummaryPrice.js

```diff
@@ -34,6 +47,37 @@ const ProductSummaryPrice = ({
     sellingPriceClass: 'dib ph2 t-body t-heading-5-ns',
   }
 
+  const sortPrices = (priceA, priceB) => priceA - priceB
+  const lowestPrice = compose(head, sort(sortPrices))
+  const highestPrice = compose(last, sort(sortPrices))
+
+  const getListPrices = prices => {
+    const lowPrice = lowestPrice(prices)
+    const highPrice = highestPrice(prices)
+    return [
+      lowPrice,
+      highPrice,
+    ]
+  }
+
+  const isAvailableProduct = price => price !== 0
+
+  const getPriceRange = () => {
+    const items = prop('items', product)
+    if (items) {
+      const sellers = flatten(pluck('sellers', items))
```

Comment:
can't this be const sellers = map(pick(['sellers']), items)?

If yes, I think this is clear and more performant thant flatten + pluck

Reply:
It works, but we have use `flatten + map`
`const sellers = flatten(map(prop('sellers'), items))`